sodapy/__init__.py

```diff
@@ -229,6 +229,40 @@ def replace(self, dataset_identifier, payload, content_type="json"):
 
         return self._perform_update("put", resource, payload)
 
+    def createNonDataSet(self, payload, files, content_type="json"):
```

Comment:
Do you know if there are any other valid keys for `payload`, other than `method`? I don't think that argument name makes sense in this context, since the actual "payload" is the file. If `method` is the only actionable keyword, then I would argue that it's actually a `content_type`, even if the request ultimately sends it as a `params` dictionary.

```
def create_nondataset(self, files, content_type="blob"):
```